components/__tests__/alarmBoolean-test.js

```diff
@@ -0,0 +1,24 @@
+import timeScreenVM from '../../viewModels/timeScreenVM/timeScreenVM'
+import {observable} from 'mobx'
+
+const timescreen = new timeScreenVM()
+
+it('returns false if no alarm set', () => {
+
+    expect(timescreen.alarmEnabled).toEqual(false)
+
+    timescreen.dispose()
+
+})
+
+it('can toggle alarm status on and off', () => {
+    timescreen.toggleAlarmEnabled()
+
+    expect(timescreen.alarmEnabled).toEqual(true)
+
+    timescreen.toggleAlarmEnabled()
+
+    expect(timescreen.alarmEnabled).toEqual(false)
+
+    timescreen.dispose()
```

Comment:
good to see you're disposing! As it's being used in two tests I would probably instantiate the timescreen a new time for each test, so move line 4 to lines 7 and 15, and duplicate line 23 on line 11